### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by lboratynski**

### CODE REVIEW

File: Translations/pl.lproj/advanced.strings [hunk 1]
```diff
@@ -15,7 +15,7 @@
 "2270.title" = "Podziel plik:";
 
 /* Class = "NSComboBoxCell"; placeholderString = "Example: 5 MB"; ObjectID = "2271"; */
-"2271.placeholderString" = "np. 5 MB";
+"2271.placeholderString" = "max. rozmiar 1 pliku - np. 5 MB";
```
Comment (aonez) [inline Translations/pl.lproj/advanced.strings:18]: @lboratynski What's the idea behind this addition?

**Comment by aonez** [comment]
Thanks a lot @lboratynski! Not minor at all 😂 Please check my comment above.

File: Translations/pl.lproj/advanced.strings (hunk 1 above)
Comment (lboratynski) [reply Translations/pl.lproj/advanced.strings:18]: In my opinion, such a notation is more clear and makes it obvious that the file will be divided into several parts. This is just a suggestion, the previous notation is also fine.

Comment (aonez) [reply Translations/pl.lproj/advanced.strings:18]: For better consistency with the rest of the translations I prefer to leave it as "np. 5 MB"

Comment (lboratynski) [reply Translations/pl.lproj/advanced.strings:18]: Sure, as you wish. That's actually the best option.

**Pull request merged**